net_transport_test.go

```diff
@@ -209,6 +211,7 @@ func TestNetworkTransport_AppendEntries(t *testing.T) {
 			},
 			LeaderCommitIndex: 90,
 		}
+		args.Addr = []byte("cartman")
```

Comment:
Not necessary, but if you are looking for a way to make this work inline in the struct definition, you can do this:

```go
RPCHeader: RPCHeader{Addr: []byte("cartman")},
```